content/zh/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -12,70 +12,108 @@ weight: 90
 <!-- overview -->
 
 <!--
-The Horizontal Pod Autoscaler automatically scales the number of Pods
-in a replication controller, deployment, replica set or stateful set based on observed CPU utilization (or, with
-[custom metrics](https://git.k8s.io/community/contributors/design-proposals/instrumentation/custom-metrics-api.md)
-support, on some other application-provided metrics). Note that Horizontal
-Pod Autoscaling does not apply to objects that can't be scaled, for example, DaemonSets.
--->
-Pod 水平自动扩缩（Horizontal Pod Autoscaler）
-可以基于 CPU 利用率自动扩缩 ReplicationController、Deployment、ReplicaSet 和
-StatefulSet 中的 Pod 数量。
-除了 CPU 利用率，也可以基于其他应程序提供的
-[自定义度量指标](https://git.k8s.io/community/contributors/design-proposals/instrumentation/custom-metrics-api.md)
-来执行自动扩缩。
-Pod 自动扩缩不适用于无法扩缩的对象，比如 DaemonSet。
-
-<!--
-The Horizontal Pod Autoscaler is implemented as a Kubernetes API resource and a controller.
+In Kubernetes, a _HorizontalPodAutoscaler_ automatically updates a workload resource (such as a
```

Comment:
In Kubernetes, a _HorizontalPodAutoscaler_ automatically updates a workload resource (such as
a {{< glossary_tooltip text="Deployment" term_id="deployment" >}} or
{{< glossary_tooltip text="StatefulSet" term_id="statefulset" >}}), with the
aim of automatically scaling the workload to match demand.